### PR TITLE
Password Generator Using React.js

### PR DESCRIPTION
**Pull request by PranavBarthwal**
# Pull Request for WebMasterLog 💡

## Issue Title: Password Generator Using React.js

- **Name of the project** :  Password-Generator
- **Category:**  React.js
- **Name:**  Pranav 
- **GitHub ID:**  https://github.com/PranavBarthwal
- **Idenitfy yourself:**  JWOC Contributor

Closes: #85 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
added random password generator project using react.

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made
by running on my local system

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

Screenshot 2024-02-14 235046

### CODE REVIEW

**Comment by PranavBarthwal** [comment]
@Avdhesh-Varshney Done the changes, hope so this PR is ready to merge!

File: React-JS-Projects/Basic/Password-Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+  ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+  <a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg" alt="react" width="40" height="40"/> </a>
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/Password-Generator/README.md:12]: use this `React` in place of react logo.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@PranavBarthwal 
That's why i told to follow the newly updated README [TEMPLATE](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/.github/PROJECT-README-TEMPLATE.md)

File: React-JS-Projects/Basic/Password-Generator/README.md (hunk 1 above)
Comment (PranavBarthwal) [reply React-JS-Projects/Basic/Password-Generator/README.md]: Done

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**